Mathlib/Algebra/Order/LatticeGroup.lean

```diff
@@ -59,12 +61,10 @@ lattices.
 lattice, ordered, group
 -/
 
-
--- Needed for squares
 -- Needed for squares
```

Comment:
What does this comment mean?

Reply:
Possibly it's some odd artefact of the porting process? I think it used to be here: https://github.com/leanprover-community/mathlib/blob/master/src/algebra/order/lattice_group.lean#L6

Reply:
Squares being used [here](https://github.com/leanprover-community/mathlib4/commit/4dced5ef925370e6ddef036201dbdb0be41f0520#diff-7f541e62bd43db13b165b8cba1216788fae6d5b2ba7fc4184af231ec348c5ac4R415) and [here](https://github.com/leanprover-community/mathlib4/commit/4dced5ef925370e6ddef036201dbdb0be41f0520#diff-7f541e62bd43db13b165b8cba1216788fae6d5b2ba7fc4184af231ec348c5ac4R425)

Reply:
It no longer seems to be necessary to explicitly import `Mathlib.Algebra.GroupPower.Basic` so I've removed both it and the comment.